book-service/src/main/java/com/jpop8/bookservice/models/BookCategory.java

```diff
@@ -0,0 +1,36 @@
+package com.jpop8.bookservice.models;
+
+import java.util.Set;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "book_category")
+@Getter
+@Setter
+public class BookCategory {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id")
+	private Long id;
+	
+	@Column(name = "category_name")
+	private String categoryName;
+	
+	@Getter(onMethod = @__( @JsonManagedReference ))
+	@OneToMany(cascade = CascadeType.ALL, mappedBy = "bookCategory")
+	private Set<Book> books;
+}
```

Comment:
Do we need a separate entity for Book Category?
I think java ENUM is sufficient.